extensions/wikia/TriviaQuizzes/js/ext.TriviaQuizzes.js

```diff
@@ -0,0 +1,25 @@
+require(['jquery', 'content_types_consumption', 'wikia.trackingOptIn', 'mw'], function ($, consumption, trackingOptIn) {
```

Comment:
`mw` should probably be on the list of method params here.

```suggestion
require(['jquery', 'content_types_consumption', 'wikia.trackingOptIn', 'mw'], function ($, consumption, trackingOptIn, mw) {
```